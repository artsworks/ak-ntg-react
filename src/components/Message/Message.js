import React from 'react';
import PropTypes from 'prop-types';

function Message(props) {
    return props.text ? (<div className="App__content-message">{props.text}</div>) : '';
}

Message.propTypes = {
    text: PropTypes.string,
}

export default Message;