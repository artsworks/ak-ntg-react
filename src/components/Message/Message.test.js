import React from 'react';
import renderer from 'react-test-renderer';
import Message from './Message';

it('renders message correctly', () => {
    const tree = renderer
        .create(<Message text='Dummy text for testing'></Message>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});