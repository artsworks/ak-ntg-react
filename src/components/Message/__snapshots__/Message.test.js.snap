// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders message correctly 1`] = `
<div
  className="App__content-message"
>
  Dummy text for testing
</div>
`;
